### DISCUSSION TITLE
ADC Buttons?

### DISCUSSION THREAD

**Discussion by Magnum8101** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2189] [Q&A]
I am trying to flash Marlin Bugfix 2.0.x On a brand new BTT SKR v1.4 Turbo with Anet A8 Configurations.

I have google different guides on internet, and watch several clips on youtube, but I can not  find any sollution for my problem when compiling.

481 |   #error "ADC BUTTONS do not work unmodified on SKR 1.4, The ADC ports cannot take more than 3.3v."

And what is ADC Buttons?

**Comment by oldman4U** [comment]
And how is this related to the BTT TFT firmware this repository is related to?

**Reply by Magnum8101** [reply]
sorry.. wrong forum... =O)
